packages/cozy-procedures/src/components/Amount.jsx

```diff
@@ -1,22 +1,74 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { withRouter } from 'react-router'
+import get from 'lodash/get'
 import Topbar from './Topbar'
-import { translate } from 'cozy-ui/transpiled/react/I18n'
+import {
+  Title,
+  translate,
+  Label,
+  Input,
+  Button
+} from 'cozy-ui/transpiled/react/'
+import creditApplicationTemplate from '../templates/creditApplicationTemplate'
```

Comment:
Just though about that, but we have to use a webpack alias to require the template